CHANGELOG.md

```diff
@@ -2,6 +2,34 @@
 * This file will be updated whenever a new release is put into production.
 * Any problems should be reported via the "report an issue" link in the footer of the application.
 
+## v4.2.0 - 2015-01-12
+### Important
+*This release fundamentally changes how Reservations is configured (see the ([wiki](https://github.com/YaleSTC/reservations/wiki/Configuration)) for more details). You will likely have to rework how Reservations is deployed and hosted for it to work properly when updating to this version.*
```

Comment:
Hmmm, stuff like this has me wondering about our versioning system. Is it really best for our major versions to track Rails major versions? Or would it be better to use Semantic Versioning, and update the major version number when we make a breaking change (like this definitely is)? I've shied away from suggesting Semantic Versioning too seriously in the past because I never really figured out if it made any sense for uh, for lack of a better word, client-facing apps like Reservations. SemVer seems more geared towards developer-facing apps / programs since it's a lot easier for those to determine what a "breaking change" actually entails (usually an API change).

But anyway, this made me think of SemVer again because we actually are introducing a breaking change here, by completely revamping configuration to the point that our clients might have to change their whole deployment process to make it work. I dunno, just something to think about.